### PR TITLE
LPS-70366 These attributes have these values by default

### PR DESCRIPTION
**Pull request by Robert-Frampton**

### CODE REVIEW

**Comment by Robert-Frampton** [comment]
Hey @jonmak08 

Things look good to me, just wanted to send it through since you worked with Support on the solution.

File: portal-web/docroot/html/common/themes/password_expiring_soon.jspf [hunk 1]
```diff
@@ -14,13 +14,13 @@
  */
 --%>
 
-<c:if test="<%= themeDisplay.isSignedIn() && UserLocalServiceUtil.isPasswordExpiringSoon(user) %>">
-	<aui:script use="liferay-notice">
-		new Liferay.Notice(
+<c:if test='<%= SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```
Comment (jonmak08) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: Hey @NolanChan and @dustinryerson,

Did we want to keep the check for ```themeDisplay.isSignedIn()``` still?  I know that ```passwordExpiringSoon``` only gets added to the ```SessionMessages``` in ```LoginPostAction```, so I was not sure if it was worthwhile to keep this check there or not.

Just let me know.  Thanks!

**Comment by dustinryerson** [comment]
ci:reopen

File: portal-web/docroot/html/common/themes/password_expiring_soon.jspf (hunk 1 above)
Comment (NolanChan) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: I was thinking about it, but my thought process assumed that we would only be in the `LoginPostAction` if we were signed in... is that correct? Also I'm not sure if I can access the themeDisplay in `LoginPostAction`, and I'm not sure how else we can access that information

Comment (jonmak08) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: I don't think you need to access ```themeDisplay``` in ```LoginPostAction```, I think you can just keep what you have and at line 17 above, you would just do:

```
<c:if test='<%= themeDisplay.isSignedIn() && SessionMessages.contains(request, "passwordExpiringSoon") %>'>
```

@dustinryerson/@holatuwol, do you have any thoughts on whether we should keep the ```themeDisplay.isSignedIn()```?

Comment (holatuwol) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: We no longer need `themeDisplay.isSignedIn()`.

Comment (dustinryerson) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: @jonmak08 looks like the source formatter is complaining about one of the changes related to this pull - I think line 110 of LoginPostAction needs one more tab.

Comment (jonmak08) [inline portal-web/docroot/html/common/themes/password_expiring_soon.jspf]: Sounds good @holatuwol!  I will update and send it along.

Please let me know if you have any questions. Thanks guys!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!
Pull request submitted at: https://github.com/brianchandotcom/liferay-portal/pull/47500

View just my changes: https://github.com/jonmak08/liferay-portal/compare/Robert-Frampton:c657cb642e...80cc2f51ef
View total diff: https://github.com/jonmak08/liferay-portal/compare/58cb1c6bbe...80cc2f51ef

**Pull request closed without merging**